src/frontends/paddle/src/op/box_coder.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs box_coder(const NodeContext& node) {
+    auto prior_box = node.get_input("PriorBox");
+    auto target_box = node.get_input("TargetBox");
+    auto axis = node.get_attribute<int32_t>("axis", 0);
+    auto norm = node.get_attribute<bool>("box_normalized", true);
+    auto uns_axes = default_opset::Constant::create(ov::element::i64, {1}, {axis});
+
+    Output<Node> variance;
+    if (node.has_input("PriorBoxVar")) {
+        variance = node.get_input("PriorBoxVar");
+        variance = std::make_shared<default_opset::Unsqueeze>(variance, uns_axes);
+    }
+    else {
```

Comment:
If without this else branch, will it be simpler in the mapped graph?

Reply:
Hi, I can't caught what you mean.
This else branch is to process the input PriorBoxVar which is a list. So I convert it to a 2-d tensor, which is the same as the input  PriorBoxVar in the if branch. So the following steps after processing are same no mater the input PriorBoxVar type is.

Reply:
sounds fine.